src/components/autosuggest-text/autossugest-text.tsx

```diff
@@ -0,0 +1,94 @@
+import { UseComboboxPropGetters } from "downshift";
+import React from "react";
+import { SuggestionsFromQueryStringQuery } from "../../core/dbc-gateway/generated/graphql";
+
+export interface AutosuggestTextProps {
+  responseData: SuggestionsFromQueryStringQuery["suggest"]["result"];
+  currentQ: string | undefined;
+  highlightedIndex: number;
+  getItemProps: UseComboboxPropGetters<Suggestion>["getItemProps"];
+  stringSuggestionAuthorText?: string;
+  stringSuggestionWorkText?: string;
+  stringSuggestionTopicText?: string;
+}
+
+interface SuggestionCreator {
+  __typename: "Creator";
+  name: string;
+}
+interface SuggestonSubject {
+  __typename: "Subject";
+  value: string;
+}
+interface SuggestionWork {
+  __typename: "Work";
+  title?: string | null;
+}
+export type Suggestion = SuggestionCreator | SuggestonSubject | SuggestionWork;
+
+export const AutosuggestText: React.FC<AutosuggestTextProps> = ({
+  responseData,
+  highlightedIndex,
+  getItemProps,
+  stringSuggestionAuthorText = "author",
+  stringSuggestionWorkText = "work",
+  stringSuggestionTopicText = "topic"
```

Comment:
We can do that later on. [#27](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/27) is not included in this branch and I would rather add that in a future feature/autosuggest-advanced branch than rebasing release/2 into this branch (since there for sure is gonna be a new branch for autosuggest)

Reply:
added it to [DDFSOEG-144](https://reload.atlassian.net/jira/software/c/projects/DDFSOEG/boards/457?modal=detail&selectedIssue=DDFSOEG-144) ticket